server2/Arcadia.Assistant/Arcadia.Assistant.Sharepoint/Sharepoint.cs

```diff
@@ -57,9 +63,24 @@ public class Sharepoint : StatelessService
             this.sickLeaves = sickLeaves;
             this.employees = employees;
             this.organizations = organizations;
+            this.notifications = notifications;
             this.serviceSettings = serviceSettings;
             this.departmentsCalendarsSettings = departmentsCalendarsSettings;
             this.logger = logger;
+            SharepointItemSynchronizationBase.SharepointItemSynchronizedEvent += this.SharepointItemSynchronizedEventHandler;
+        }
+
+        private void SharepointItemSynchronizedEventHandler(object sender, SharepointItemSynchronizationBase.SharepointSynchronizationArgs e)
```

Comment:
let's not make changes to sharepoint now. It's external storage, no way it should be sending those notifications. Also, keep in mind that not all departments have those calendars. Each service will have to send its own notifications.

Reply:
I add TODO for future fix it.